### PR TITLE
one more exclude; rm pkg versions from DESC

### PR DESCRIPTION
**Pull request by pawelru**
- I missed one more exclude
- `>= x.y.z.900x` prevents from rendering within stable profile - removed all pkgs versions

### CODE REVIEW

**Comment by Melkiades** [comment]
> * `>= x.y.z.900x` prevents from rendering within stable profile - removed all pkgs versions

Is this ok to do? I guess it is the stable so it should be fine anyway right?

**Comment by pawelru** [comment]
> > * `>= x.y.z.900x` prevents from rendering within stable profile - removed all pkgs versions
> 
> Is this ok to do? I guess it is the stable so it should be fine anyway right?

This is ok as long as such condition is met for both stable and devel profile - right now that's not the case due to condition on `tern`. I think we should do individual article / code chunk exclusions instead of introducing this at the package level which essentially could block the whole test.

**Review by Melkiades — APPROVED** [review]
Lgtm! Thanks Pawel!!! ^^

File: package/DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,6 @@
 Package: tlg.catalog.pkg
 Title: Tables, Listings and Graphs Catalog
 Version: 0.1.0.9009
-Date: 2024-01-26
```
Comment (Melkiades) [inline package/DESCRIPTION:4]: Just noticed you removed also Date... Was it also necessary?

Comment (pawelru) [reply package/DESCRIPTION:4]: R CMD CHECK gave me note about it so I simply removed this :) It's not super critical

**Pull request merged**